elastic/kibana
# [Response Ops][Alerting] Classify `index pattern not found` errors from ES query rule as user error

Resolves https://github.com/elastic/response-ops-team/issues/220

## Summary

Classifies errors caused by users deleting data view used by the ES query KQL rule as user errors.

## To Verify
1. Create a KQL ES query rule that uses a data view and let it run successfully.
2. Delete the data view
3. See the rule execution fail and check the metrics from https://localhost:5601/api/task_manager/metrics?reset=false classify the error as a user error (in `metrics.task_run.value.overall.user_errors`)
